ccp/modules/datashield-setup.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+if [ "$ENABLE_DATASHIELD" == true ]; then
+  log INFO "DataSHIELD setup detected -- will start DataSHIELD services."
+  OVERRIDE+=" -f ./$PROJECT/modules/datashield-compose.yml"
+  OPAL_DB_PASSWORD="$(echo \"This is a salt string to generate one consistent password for Opal. It is not required to be secret.\" | openssl rsautl -sign -inkey /etc/bridgehead/pki/${SITE_ID}.priv.pem | base64 | head -c 30)"
+  DATASHIELD_CONNECT_SECRET="$(echo \"This is a salt string to generate one consistent password as the DataShield Connect secret. It is not required to be secret.\" | openssl rsautl -sign -inkey /etc/bridgehead/pki/${SITE_ID}.priv.pem | base64 | head -c 30)"
+  if [ ! -e /tmp/bridgehead/opal-cert.pem ]; then
+    mkdir -p /tmp/bridgehead/
+    chown -R bridgehead:docker /tmp/bridgehead/
+    openssl req -x509 -newkey rsa:4096 -nodes -keyout /tmp/bridgehead/opal-key.pem -out /tmp/bridgehead/opal-cert.pem -days 3650 -subj "/CN=${HOST:-opal}/C=DE"
+    chmod g+r /tmp/bridgehead/opal-key.pem
+  fi
+  mkdir -p /tmp/bridgehead/opal-map
+  jq -n --argfile input ./$PROJECT/modules/datashield-mappings.json '
```

Comment:
This call adds a new dependency to the bridgehead: "jq"
With this it is necessary to install jq on all bridgehead servers before enabling datashield, otherwise bridgeheads will crash and not start

Reply:
I think that @Threated has already done it. Isn't is @Threated ?

Reply:
Yes we use dockerized jq now